### PR TITLE
top 6 tools for dev productivity in 2022

### PR DESCRIPTION
**Pull request by acoudouy**

### CODE REVIEW

**Comment by acoudouy** [comment]
image

File: data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  classNameImage=""
+  width={1200}
+  height={796}
+/>{" "}
+
+It's an ideal companion for your Mac. Raycast is a productivity app that lets you access all of your apps, files, and services from a single location. It's a powerful tool for searching for files, launching apps, and performing calculations. It also includes a terminal that allows you to perform commands and view the results. Raycast is a powerful tool for searching for files, launching apps, and performing calculations. It also includes a terminal that allows you to perform commands and view the results.
+
+To be honest, I'm dying to try it. I'm a Windows user (yes, that exists) and my friend Sydney speaks about it whenever he can.
+
+### <NoFollowLink url="https://justgetflux.com/">F.lux</NoFollowLink> - to protect your eyes from screen fatigue
```
Comment (cosydney) [inline data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx:106]: c'est developer comfort ca ^^

**Comment by acoudouy** [comment]
image

File: data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx [hunk 2]
```diff
@@ -0,0 +1,123 @@
+---
+title: "Best 6 tools to improve developer productivity in 2022"
+date: "2022-09-22"
+tags: ["software-development", "developer-productivity"]
+draft: false
+summary: "Hi! And welcome to our list of best developer tools for productivity. You will find 6 tools to help your day-to-day as a developer, from improving PR review time, passing through maintaining your flow, to the best software for pair programming."
```
Comment (cosydney) [inline data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx:6]:
```suggestion
summary: "You will find 6 tools to help your day-to-day as a developer, from improving PR review time, passing through maintaining your flow, to the best software for pair programming."
```

**Review by cosydney — APPROVED** [review]

File: data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx (hunk 1 above)
Comment (acoudouy) [reply data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx:106]: confort = pouvoir passer plus de temps sur l'ordi pour moi ici

File: data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx (hunk 2 above)
Comment (cosydney) [reply data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx:6]: c’est ce que les gens vont voir dans les resultat de google je me dit autant aller straight to the point [](edited-with-Axolo)

Comment (acoudouy) [reply data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx:6]: Ah yes j'avais pas capté que c'était le summary. je vais l'améliorer [](edited-with-Axolo)

Comment (acoudouy) [reply data/p/best-6-tools-to-improve-developer-productivity-in-2022.mdx:6]: If you want to have the feeling to code more or to code as much as today but wish finish earlier, houray! You will find in this article 6 tools to help your day-to-day as a developer, from improving PR review time, passing through maintaining your flow, to the best software for pair programming. [](edited-with-Axolo)

**Pull request merged**